<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Countries</class>
 <widget class="QDialog" name="Countries">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>476</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>476</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>476</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lwServers">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>321</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leServers">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Your Server IP: 127.0.0.1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChoose">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
